source/help/submit/index.md

```diff
@@ -50,6 +49,25 @@ We do not accept [dvi, PS, or PDF created
 from TeX/LaTeX source](../faq/whytex.md), and we
 do not accept scanned documents, regardless of format.
 
+
+## (La)TeX processing changes &mdash; April 2024
+
+We will soon be rolling out changes to how arXiv process (La)TeX subbmissions. These changes shouldn't be noticable to most of our users. We will be retiring the arXiv-developed "AutoTeX' system that we've used for decades in favor of a simpler, more straight-foward process of convering (La)TeX submissions to PDFs..
+
+ 1. The AutoTeX would try different versions of TeX to see which one successfully builds a PDF. Going forward we will only use the version of TeX currently used by arXiv
+    - Our plan is that arXivs "current" version closely tracck the annual TeX Live releases. In the past, we've often gone a few years between TeX updates
+ 1. The AutoTeX system would attempt to detect image files that were not referenced in the (La)TeX source for the paper and append those images to the end of the PDF for the paper. Based on recent feedback, we find that most authors are surprised by this behvior, and we will thus be discontinuing this practice. If an author wants images at the end of their paper, just use the normal TeX procedures for this (see https://latex-tutorial.com/tutorials/figures/).
+1. The AutoTeX system would apply the [LaTeX hyperref package](https://ctan.org/pkg/hyperref) to LaTeX submissions that did not include `\use {hyperref}`. We will no longer do that. (Note &ndash; before you add a `\use {hyperref}` to your main.tex, check to see if the template you're using already has a reference; most do).
+1. The AutoTeX system would look for any string in a paper that looked like a reference to an arXiv paper (such as 	arXiv:2402.08954, or even just 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
```

Comment:
I think AutoTeX does that only in the references, **not** in the complete text of the article.